models/speedmodes.js

```diff
@@ -28,13 +28,15 @@ module.exports = {
     { homekitTopLevel: 100, miLevel: 100, name: "Max" }
   ],
   gen3: [
-    // 0%      = Off / Aus
+    // 0% = Off / Aus
     { homekitTopLevel: 0, miLevel: 0, name: "Off" },
-    // 1-38%   = "Quiet / Leise"
+    // 1-15% = "Soft"
+    { homekitTopLevel: 15, miLevel: 105, name: "Soft" },
```

Comment:
Adding this mode to `gen3`, it makes it exactly the same as `gen4`. Shouldn't we simply change the model to point to `gen4`?